sql/storm-sql-core/src/test/org/apache/storm/sql/TestStormSql.java

```diff
@@ -29,13 +29,14 @@
 import org.apache.storm.sql.runtime.ISqlStreamsDataSource;
 import org.apache.storm.streams.Pair;
 import org.apache.storm.tuple.Values;
-import org.junit.Assert;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
```

Comment:
can this be removed, since static method is being imported below?

Reply:
done